### PR TITLE
Move some deprecations along.

### PR DESCRIPTION
**Pull request by jmdavis**

### CODE REVIEW

File: std/bitmanip.d [hunk 1]
```diff
@@ -1237,17 +1237,19 @@ public:
...
-    deprecated("Please use the constructor instead. This function will be removed in Jan 2016.")
-    /***************************************
-     * $(RED Will be deprecated in 2.068. Please use the constructor instead.)
-     */
+    // @@@DEPRECATED_2016-01@@@
+    /++
+        $(RED Deprecated. Please use the constructor instead. This will be
+              removed in January 2016.)
+      +/
+    deprecated("Please use the constructor instead.")
```
Comment (burner) [inline std/bitmanip.d]: IMO the "please" should be removed. It just sounds weak.

Comment (jmdavis) [inline std/bitmanip.d]: Hmmm. Maybe it would be better to not use please, but almost all of the deprecation messages currently do.

Comment (burner) [inline std/bitmanip.d]: Clearly there is no correct choice then, do whatever you think is correct.

Comment (jmdavis) [inline std/bitmanip.d]: Well, I may change the messages at some point (if nothing else, to make them shorter), but if I do, I might as well change them all at once. I'll them as-is for the moment.

Comment (burner) [inline std/bitmanip.d]: Sounds like a plan

File: std/functional.d [hunk 2]
```diff
@@ -138,13 +138,6 @@ unittest
     assert(isEven(2) && !isEven(1));
 }
 
-/+ Undocumented, will be removed December 2014+/
```
Comment (Hackerpilot) [inline std/functional.d]: Ha.

**Comment by Hackerpilot** [comment]
:white_check_mark:

**Comment by burner** [comment]
LGTM

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

**Pull request merged**